backend/src/modules/image/image.controller.ts

```diff
@@ -0,0 +1,29 @@
+import {
+    Controller,
+    Post,
+    Res,
+    UploadedFile,
+    UseGuards,
+    UseInterceptors,
+} from '@nestjs/common';
+import { ImageService } from './image.service';
+import { JwtGuard } from '../../guard/jwt.guard';
+import { FileInterceptor } from '@nestjs/platform-express';
+import { ApiOperation, ApiResponse } from '@nestjs/swagger';
+import { UploadImage } from './swagger';
+
+@Controller('image')
+export class ImageController {
+    constructor(private readonly imageService: ImageService) {}
+
+    @ApiOperation(UploadImage.Operation)
+    @ApiResponse(UploadImage._201)
+    @ApiResponse(UploadImage._401)
+    @UseGuards(JwtGuard)
+    @Post('upload')
+    @UseInterceptors(FileInterceptor('file'))
+    async uploadImage(@UploadedFile() file, @Res() res) {
+        const imageURL = await this.imageService.create(file);
+        return res.send({ imageURL });
```

Comment:
```suggestion
    async uploadImage(@UploadedFile() file) {
        const imageURL = await this.imageService.create(file);
        return { imageURL };
```
응답을 위해 `@Res` 객체를 사용하신 게 아니라면 res는 없어도 좋을 것 같습니다!